content/developer/reference/frontend.rst

```diff
@@ -15,7 +15,6 @@ JavaScript framework
     frontend/services
     frontend/hooks
     frontend/patching_code
-    frontend/javascript_cheatsheet
```

Comment:
This will trigger a 404 for readers browsing the page or having bookmarked it. Consider [adding a new redirect rule](https://github.com/odoo/documentation/blob/16.0/redirects/MANUAL.md) targeting the new `javascript_fields` howto.